frontend/dispatcher.lua

```diff
@@ -129,6 +129,7 @@ local settingsList = {
     book_cover = {category="none", event="ShowBookCover", title=_("Book cover"), reader=true, separator=true},
     show_config_menu = {category="none", event="ShowConfigMenu", title=_("Show bottom menu"), reader=true},
     toggle_bookmark = {category="none", event="ToggleBookmark", title=_("Toggle bookmark"), reader=true},
+    toggle_page_change_Animation = {category="none", event="TogglePageChangeAnimation", title=_("Toggle page turn animations"), reader=true, condition=Device:canDoSwipeAnimation()},
```

Comment:
It doesn't affect functionality, but I just noticed it says `toggle_page_change_Animation` instead of the expected `toggle_page_change_animation`.

(See https://www.mobileread.com/forums/showthread.php?p=4227235#post4227235 as to why I'm looking.)